aiozk/connection.py

```diff
@@ -4,11 +4,13 @@
 import re
 import struct
 import sys
+from time import time
 
 # from tornado import ioloop, iostream, gen, concurrent, tcpclient
 
 from aiozk import protocol, iterables, exc
 
+DEFAULT_READ_TIMEOUT = 1
```

Comment:
I believe we should use bigger default timeout value. Maybe 3 seconds would be better